R/foreign.R

```diff
@@ -622,10 +623,10 @@ graph_from_graphdb <- function(url = NULL,
     typegroup <- typegroups[which(types == type)]
 
     if (!prefix %in% prefixes) {
-      stop("Invalid prefix!")
+      cli::cli_abort("{prefix} is not a valid prefix. Must be one of {prefixes}.")
     }
     if (!type %in% types) {
-      stop("Invalid graph type!")
+      cli::cli_abort("{type} is not a valid graph type. Must be one of {types}.")
```

Comment:
maybe split this and the message above into two lines?
```suggestion
      cli::cli_abort(c(
      "{type} is not a valid graph type.",
      i = "Must be one of {types}."
    ))
```